packages/contracts/contracts/interfaces/IRecipientRegistry.sol

```diff
@@ -15,9 +15,9 @@ interface IRecipientRegistry {
   }
 
   /// @notice Events
-  event RecipientAdded(uint256 indexed index, bytes32 id, bytes32 indexed metadataUrl, address indexed recipient);
-  event RecipientRemoved(uint256 indexed index, bytes32 id, address indexed recipient);
-  event RecipientChanged(uint256 indexed index, bytes32 id, bytes32 indexed metadataUrl, address indexed newAddress);
+  event RecipientAdded(uint256 indexed index, bytes32 id, bytes32 indexed metadataUrl, address indexed payout);
```

Comment:
I think `payout` works for both, doesn't matter if it's funding or not. If it's non funding, I think it can be just zero address. Btw, it was `recipient` before but I was confused about `id` and `recipient` and I change it to `payout` to make it more generic.